plugin/completion.py

```diff
@@ -172,16 +218,21 @@ def _resolve_completions_async(self, responses: List[ResolvedCompletions]) -> No
             response_items = []  # type: List[CompletionItem]
             if isinstance(response, dict):
                 response_items = response["items"] or []
+                item_defaults = response.get('itemDefaults') or {}
                 if response.get("isIncomplete", False):
                     flags |= sublime.DYNAMIC_COMPLETIONS
             elif isinstance(response, list):
                 response_items = response
             response_items = sorted(response_items, key=lambda item: item.get("sortText") or item["label"])
-            LspSelectCompletionCommand.completions[session.config.name] = response_items
+            LspSelectCompletionCommand.completions[session.config.name] = {
+                'items': response_items,
+                'itemDefaults': item_defaults
+            }
             can_resolve_completion_items = session.has_capability('completionProvider.resolveProvider')
             config_name = session.config.name
             items.extend(
-                format_completion(response_item, index, can_resolve_completion_items, config_name, self._view.id())
+                format_completion(response_item, index, can_resolve_completion_items,
+                                  config_name, item_defaults, self._view.id())
```

Comment:
prefer this formatting:
```suggestion
                format_completion(
                    response_item, index, can_resolve_completion_items, config_name, item_defaults, self._view.id())
```